docs/core/tools/dotnet-test.md

```diff
@@ -11,18 +11,24 @@ ms.date: 03/27/2024
 
 ## Description
 
-The `dotnet test` command builds the solution and runs the tests with either VSTest or Microsoft Testing Platform (MTP). To enable MTP, you need to add a config file named `dotnet.config` with an INI-like format located at the root of the solution or repository.
+The `dotnet test` command builds the solution and runs the tests with either VSTest or Microsoft Testing Platform (MTP). To enable MTP, you need to specify the test runner in the `global.json` file.
 
-Some examples of the `dotnet.config` file:
+Some examples of how to specify the test runner in the [`global.json`](global-json.md) file:
 
-  ```ini
-  [dotnet.test.runner]
-  name = "Microsoft.Testing.Platform"
+  ```json
+  {
+      "test": {
+          "runner": "Microsoft.Testing.Platform"
+      }
+  }
   ```
 
-  ```ini
-  [dotnet.test.runner]
-  name = "VSTest"
+  ```json
+  {
+      "test": {
+          "runner": "VSTest"
+      }
+  }
   ```
```

Comment:
I'm not sure if it's worth showing this, as VSTest is already the default. So people modifying their global.json will likely be using MTP.